### ISSUE TITLE
Typo in Outlet Name

### ISSUE THREAD

**Issue by brooks42** [holojack/CoinAlert-MacOS#1]
One of the outlet properties has a typo in its name.          

```
            <connections>
                <outlet property="quitClickecd" destination="GNu-fN-BRG" id="tGp-2a-ERD"/>
                <outlet property="statusMenu" destination="7rZ-hl-dbs" id="YBN-3C-Jae"/>
            </connections>
```

**Comment by d1str0** [comment]
#2 fixes this by removing it

**Comment by hpassmore** [comment]
Fixed in commit 43b50d6
